building/configlet/README.md.new

```diff
@@ -0,0 +1,40 @@
+# Configlet
```

Comment:
@axtens I think this file got added by mistake.

Reply:
Definitely. A spell artifact that should have been deleted

Reply:
Can you push an extra commit with it removed please?

Reply:
I have tried to do that. I'm not at all sure if successfully.